### PR TITLE
[12.0][ADD] hr_expense_tier_validation

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module extends the functionality of Expense Reports to support a tier validation process.

### CODE REVIEW

File: hr_expense_tier_validation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    'license': 'AGPL-3',
+    'website': 'https://github.com/OCA/hr',
+    'depends': [
+        'hr_expense',
+        'base_tier_validation',
+    ],
+    'data': [
+        'views/hr_expense_sheet_view.xml',
+    ],
+    'installable': True,
```
Comment (kittiu) [inline hr_expense_tier_validation/__manifest__.py:19]: Please add yourself as maintainers

**Review by kittiu — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Could you please rebase your branch? that would fix runbot.

**Comment by LoisRForgeFlow** [comment]
To rebase properly you also need to fetch the oca remote first, if you navigate your branch you will see that is not up-to-date (your most recent commit appart from this PR's is from June).

**Comment by LoisRForgeFlow** [comment]
Superseded by https://github.com/OCA/hr/pull/696

**Pull request closed without merging**